x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/saml/SamlMetadataCommandTests.java

```diff
@@ -569,6 +569,7 @@ public void testSigningMetadataWithPasswordProtectedPemInTerminal() throws Excep
         assertThat(validateSignature(descriptor.getSignature()), equalTo(true));
     }
 
+    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/75097")
```

Comment:
FWIW I'd be fine with us changing the certificates in these tests instead of muting them. Supporting CA certificates was never an intentional thing and there is little value in randomly testing for it. If we want to test that CA certificates can be used for other things than HTTP TLS , let's add specific tests for it to be sure. (In a follow up that we can track in an issue that is )

Reply:
Nevertheless, thanks to those tests we discovered potential breaking change we may introduce, I would leave them as they are.

Reply:
Yeap, but it was just luck that we did discover them because someone at some point created a CA certificate for SAML tests (accidentally). What I am suggesting is to actually test for it since we want to support it and we now know it

Reply:
I will add dedicated tests once we have the fix to limit filtering to tls keystore only